meta-oe/recipes-devtools/uftrace/uftrace_0.15.2.bb

```diff
@@ -1,4 +1,4 @@
-SUMMARY = "Trace and analyze execution of a program written in C/C++"
+SUMMARY = "Trace and analyze execution of a program written in C, C++, Rust and Python"
```

Comment:
I have picked this change separately, rest of this recipe upgrade it already staged in `master-next` branch

Reply:
Thank you @kraj